Gameplay/CutsceneManager.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include <scripting\Script.h>
```

Comment:
```suggestion
#include <scripting/Script.h>
```
You should change this btw.